File: app/src/main/java/danb/speedrunbrowser/api/objects/GameRuleset.java
package danb.speedrunbrowser.api.objects;

import java.io.Serializable;

class GameRuleset implements Serializable {
    public boolean showMilliseconds;
    public boolean requireVerification;
    public boolean requireVideo;
    public String[] runTimes;
    public String defaultTime;
    public boolean emulatorsAllowed;
}

File: app/src/main/java/danb/speedrunbrowser/api/objects/GameRuleset.kt
package danb.speedrunbrowser.api.objects

import java.io.Serializable

data class GameRuleset(
    var showMilliseconds: Boolean = false,
    var requireVerification: Boolean = false,
    var requireVideo: Boolean = false,
    var runTimes: Array<String>? = null,
    var defaultTime: String? = null,
    var emulatorsAllowed: Boolean = false
) : Serializable
